wordpress-mobile/WordPress-Android
# [Dependency Updates] Remove WireMock Mock Version

Parent #17558

This `2.23.2` version of `WireMock` specific to the `mocks` module is a discrepancy that should be aligned with the main `WireMock` version used on all other modules.

PS: Actually, this `2.23.2` version is anyway being updated to the main `2.26.3` version due to transitive dependencies. As such, this versioning related configuration change shouldn't matter much.

-----

PS: @zwarm I added you as the main reviewer, that is, in addition to the @wordpress-mobile/apps-infrastructure team itself, but randomly, since I just wanted someone from the `WordPress` team to sign-off on that change for WPAndroid.

-----

To test:

1. There is nothing much to test here.
2. Verifying that all the CI checks are successful should be enough.
3. However, if you want to be thorough about reviewing this change, you could see the dependency tree diff result and verify correctness.

-----

## Regression Notes

1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

See `To test` section above.

3. What automated tests I added (or what prevented me from doing so)

N/A

-----

PR submission checklist:
